node/node.go

```diff
@@ -44,9 +45,14 @@ const (
 	JoinTypeOTAA = "OTAA"
 	// JoinTypeABP is the ABP Join type.
 	JoinTypeABP = "ABP"
+	// DownlinkKey is the DoCommand Key to send a payload to the gateway.
+	DownlinkKey = "send_downlink"
+	// GatewaySendDownlinkKey is the DoCommand Key for the gateway model to queue a downlink.
+	GatewaySendDownlinkKey = "add_downlink_to_queue"
```

Comment:
@oliviamiller instead of using `senddown` on the gateway, I changed the key to be this. can easily switch it back if we prefer the smaller message, but I kinda prefer the more verbose response to the user